Knoema.Client/Search/TimeseriesSearch/GeographyRegion.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace Knoema.Search.TimeseriesSearch
+{
+	public class GeographyRegion
+	{
+		public string Id { get; set; }
+		public string LocalizedName { get; set; }
+		public string Name { get; set; }
+		public IEnumerable<string> Parents { get; set; }
+	}
+}
```

Comment:
Also, we usually end files with an empty line. You can see git is also showing warning there.

Reply:
done